src/main.c

```diff
@@ -33,31 +37,48 @@ int main(int argc, char **argv)
 
         user_input = getstr();
         if(feof(stdin) || ferror(stdin)) goto cleanup;
-
         command = strtok(user_input, " ");
 
-             if(!command) goto loop_cleanup;
+            p=strtok(NULL, "");
+            c=0,b=0,d=0;
+            while(*(p+b)==' ') b++;
+            while(*(p+c+b)!='\0')
+            {
+                *(p+c)=*(p+c+b);
+                c++;
+            }
+
+            while(*(p+d)!='\0')
+            {
+                if(*(p+d)!=' ')
+                    l=d;
 
+                d++;
+            }
+            *(p+l+1)='\0';
```

Comment:
Can we use the box bracket syntax as used elsewhere in the code, for clarity and uniformity?